virttest/utils_misc.py

```diff
@@ -4024,26 +4025,36 @@ def get_model_features(model_name):
     """
     features = []
     conf = "/usr/share/libvirt/cpu_map.xml"
+    conf_dir = "/usr/share/libvirt/cpu_map/"
```

Comment:
```python
conf = "/usr/share/libvirt/cpu_map.xml"
# check the libvirt version to branch out instead of path
if libvirt_version.version_compare(5, 0, 0):
    conf = "/usr/share/libvirt/cpu_map/"
    # we should raise the error if the path doesn't available
    if not os.path.exists(conf):
        test.error()
    with open(conf, 'r') as output:
        :::
else:
    :::
```

Reply:
@balamuruhans 
Thank you very much for pointing out my code errors.

Reply:
The utils_misc.py  file path is "~/avocado-vt/virttest/utils_misc.py"
The libvirt_version.py  file path is "tp-libvirt/provider/ libvirt_version.py "
I can not import  libvirt_version.version_compare()  directly in this  utils_misc.py.
Should  I write a function to get libvirt version, or what  can I do so that I can import the  libvirt_version.version_compare() ?

Reply:
I would suggested to move the code from `tp-libvirt/provider/libvirt_version.py` to `avocado-vt/virttest/` and use it.

Reply:
@dzhengfy @sathnaga request for your acknowlegement.

Reply:
looks like moving `libvirt_version.py` needs a change in many tests, for now we can copy `libvirt_version.py` to virttest and use in this testcase and slowly change import in other testcases through different PR and once all movement happened, we can safely remove `tp-libvirt/provider/libvirt_version.py`

Reply:
yes, once moving to virttest, we will have to send separate PR to `tp-libirt` to adopt it by changing all the testcases using it from virttest and remove the file in `provider`